### DISCUSSION TITLE
Any plan to support Qwik?

### DISCUSSION THREAD

**Discussion by cesarve77** [TanStack/query#5700] [Ideas]
I was reading about Qwik i seems very impressive the only missing part is qwik-query
https://github.com/BuilderIO/qwik/issues/4780

**Comment by gioboa** [comment]
I'm working to drop a PR for this

**Comment by md-owes** [comment]
I am exploring Qwik for my next project/product. Right now have integrated axios but would prefer TanStack when available

**Reply by TkDodo** [reply]
- https://github.com/TanStack/query/pull/5809

**Reply by md-owes** [reply]
Let me know if there is any support that i can provide to move a bit faster on that MR

**Reply by gioboa** [reply]
I need to replace few any with the right type and then the PR it's ready for the first release.

**Comment by FedeRBeije** [comment]
Hy guys, i'm starting a new project with `Qwik city`, i do not understand if it is possible to use `tanstack-query`(and others tanstack lib)
or it is still not supported.

Thank you so much for you

**Reply by gioboa** [reply]
Unfortunately it's still not available. I stopped my attempt to create the integration because I faced few TypeScript issues. 😔

**Reply by FedeRBeije** [reply]
Can i help in any way? it would be my first time and it would be a pleasure

**Reply by TkDodo** [reply]
@gioboa @FedeRBeije I would love if we could revive the qwik-query adapter and get it over the finish line 🙌

**Reply by FedeRBeije** [reply]
If you tell me how should i move and what to do i can start help on Monday :)

**Reply by FedeRBeije** [reply]
Are any of the tanstack libraries supporting qwik right now?

**Reply by gioboa** [reply]
No, there aren't, but you can use [this repo](https://github.com/gioboa/qwik-tanstack) to start digging into the integration if you want. You need to update the dependencies because it's quite old but it's a good starting point I guess

**Reply by FedeRBeije** [reply]
Okay thanks, any advance on how can i start to help you? any specific files or i don't know. Sorry for the stupid questions

**Reply by gioboa** [reply]
You can fork the repo, update the dependencies and try to reproduce the same things you are doing with other frameworks starting from `useQuery` I guess 
In this [folder](https://github.com/gioboa/qwik-tanstack/tree/main/src/qwik-query) you have my attempts to create the integration
